more java 8 removal (#5261)
jenkins-infra/jenkins.io: content/doc/administration/requirements/upgrade-java-guidelines.adoc

If you use Docker containers to run Jenkins, the default Docker containers use Java 11 beginning with link:/doc/upgrade-guide/2.303/#upgrading-to-jenkins-lts-2-303-1[Jenkins 2.303.1].
If you need to remain with Java 8 in a Docker container, append `-jdk8` to the Docker image tag.

> Starting with the 2.357 weekly release, Java 11 will be required for weekly releases.

Reviewer: ```suggestion
Starting with the 2.357 weekly release, Java 11 is required for weekly releases.
```
